### PR TITLE
Prism7 wpf

### PR DESCRIPTION
**Pull request by MichaelPonti**
Update of the introduction.md and initializing.md files in wpf for Prism 7.

### CODE REVIEW

**Comment by brianlagunas** [comment]
Thanks for the PR again :)

Like I said in the last PR you submitted, it's greatly appreciated, but we are archiving the current WPF docs and writing new topics for v7. Trying to make it more feature focused and not such a wall of text.

**Comment by MichaelPonti** [comment]
Sorry Brian, I only remember that you didn't want to have a separate wpf6 and wpf7 section, but that you liked the changes to introduction.md and initializing.md. 

I would still like to try and help, if you want, but if you have something else already in place or being worked on, I don't want to duplicate the effort.

Thanks
Mike

**Comment by brianlagunas** [comment]
Yeah, it was a decision made recently. The current WPF docs are way to wordy, not organized well, and are hard to follow.  So I will just tuck these away in a "Legacy" folder, and then start stubbing out new topics. Then slowly start filling out the content.

**Comment by brianlagunas** [comment]
Oh, and once we have all the new v7 docs in place, I will delete the "Legacy" folder.

**Comment by MichaelPonti** [comment]
Thanks Brian. If you want some help on filling out the stub content once they are in place and organized how you want, reach out and let me know.

**Comment by brianlagunas** [comment]
Definitely! Just keep an eye out for the new topics and then we can start picking topics to start filling out
